website/docs/cairo-1/05-testing/01-unit-testing.md

```diff
@@ -0,0 +1,63 @@
+---
+sidebar_label: Unit testing
+---
+
+# Unit testing
+
+
+## Writing your first test (TODO test if the example works)
+
+First, add to the file `src/lib.cairo` following code:
+```
+fn sum(a: felt252, b: felt252) -> felt252 {
+    return a + b;
+}
+```
+
+Now, let's test this function. Create a file `tests/test_sum.cairo`:
+```
+use your_project_name::sum
+
+#[test]
+fn test_sum() {
+    assert(sum(2, 3) == 5, 'sum incorrect');
+}
+```
+
+Now run your test using command:
```

Comment:
```suggestion
Now run your test using this command:
```

Reply:
Unresolving, there should definitely be an article before the word "command"

Reply:
@piotmag769 I added it. I use resolve to mark what I did when I do bulk changes